docs/cimpl6ToolingReference.md

```diff
@@ -242,31 +244,18 @@ These configurations are used to control the production of the IG. The contents
 | `showPrimaryOnly`  | `boolean` | If `showPrimaryOnly` is true, then only those profiles listed in the Content Profile (i.e., the primary profiles) will be shown on the Profiles page of the IG, and supporting profiles (i.e., the profiles _referenced by_ the primary profiles) will not be shown. Otherwise, all profiles (primary and supporting) will be listed. |
 |`changesLink`  |`string` |The URL to a site where users can request changes (shown in page footer) **(TO DO: clarify where and how this is used)** |
 |`primarySelectionStrategy`|`{}`     | _Deprecated after SHR-CLI 6.7.0_. The strategy for selection of what is primary in the IG ([see below](#primary-selection-strategy-configuration-paramters)). |
+|`"dependencies"`|`{}`     | Valid as of SHR-CLI 6.9.0. A list of other IGs that this guide depends on, allowing maps to dependent IG elements. ([see below](#Dependencies)). |
```

Comment:
Since nothing else here is in quotes, I don't think "dependencies" should have quotes. I also think it might be more clear to say "maps to external IG elements", since the elements we are mapping to are not dependent, they are the being depended upon by our internally defined elements.